syntaxes/vhdl.tmLanguage.yml

```diff
@@ -721,7 +722,7 @@ repository:
           (\=\>)
         captures:
           1: { name: keyword.reserved.vhdl }
-          2: { name: variable.other.constant.vhdl }
+          2: { name: support.constant.vhdl }
```

Comment:
The reason for his change is because the textmate , [guidlines ](https://macromates.com/manual/en/language_grammars) state that constants provided by a library should be defined as "support.constant"

It could potentially be scoped as constant.language since they are technically part of the standard.

Reply:
For a time I think the IEEE ones were separate, but either way I agree with this change in the context of the textmate guidelines.